pkg/kubelet/dockershim/docker_container.go

```diff
@@ -243,10 +239,12 @@ func (ds *dockerService) createContainerLogSymlink(containerID string) error {
 			glog.V(10).Infof("Docker logging driver is %s", dockerLoggingDriver)
 		}
 
-		if dockerLoggingDriver == dockerDefaultLoggingDriver {
-			glog.Errorf("Cannot create symbolic link because container log file doesn't exist!")
-		} else {
-			glog.V(5).Infof("Unsupported logging driver: %s", dockerLoggingDriver)
+		for _, driver := range criSupportedLogDrivers {
+			if dockerLoggingDriver == driver {
+				glog.Errorf("Cannot create symbolic link because container log file doesn't exist!")
```

Comment:
I remember the problem is that some `InspectContainer` of fake docker client doesn't return the log path.
If we return error here, it will break many unit tests.

And it's fine to not return error here, because:
1) In real case, real path should not be never be nil;
2) The symlink is a best effort thing now. Even if we return an error here, next `SyncPod` will think the container is running, because the container is started already.